### DISCUSSION TITLE
Github landing page

### DISCUSSION THREAD

**Discussion by Menace723** [community/community#41683] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i have published a project to github but upon using the link through github. None of the pictures on my page shows up. How can i fix this?

**Comment by radar-blip** [comment, marked as answer]
According to your index.html, the image links point to a `photos` folder.                
`<img class="foodndrinks" src="../photos/moms-beers.jpg" alt="beer bootles"><h3>BEERS</h3>`

But, there is no `photos` folder (or images) in your repository, so there are no images to be shown.
You need to upload the images into a folder called `photos`

**Reply by Menace723** [reply]
wow, thank you so much. i feel so dumb hahaha

**Reply by Menace723** [reply]
Hey again, so i made the folder, everything seem to worked out except the background image. i cant figure out whats going on with it.

**Reply by radar-blip** [reply]
The background with the trees in it? That seems to be working.

**Reply by Menace723** [reply]
It seems I was too impatient, sorry about that. Thank you for your time !

**Reply by radar-blip** [reply]
Its ok, not a problem.

**Reply by MatthewProjects** [reply]
I cant get my page to publish can someone help me I think there is something wrong with one of my files or my root is messed up and im getting the cant find file error 404.

**Reply by radar-blip** [reply]
@MatthewProjects Which repository is your page in?

**Reply by MatthewProjects** [reply]
**[https://github.com/MatthewProjects/CF-Portfolio](url)**

**Reply by radar-blip** [reply]
You will need to change your `Index.html` to `index.html` for Github Pages to recognize it.
Then it should work.

**Reply by MatthewProjects** [reply]
https://github.com/desktop/desktop/issues/5866

**Reply by MatthewProjects** [reply]
> You will need to change your `Index.html` to `index.html` for Github Pages to recognize it.
> Then it should work.



YAYAYAY!!!!!! that worked !!! thanks so much!!!

I still do not know why GitHub desktop is not recognizing VS code... do you know why that could be ????

**Reply by MatthewProjects** [reply]
my page [:](https://matthewprojects.github.io/CF-Portfolio/) now fully works but GitHub desktop is not seeing or recognizing VS code.......

**Reply by MatthewProjects** [reply]
https://matthewprojects.github.io/CF-Portfolio/

**Reply by radar-blip** [reply]
I have no idea why VS isnt working for you. I don't use that so I couldn't tell you.

**Comment by MatthewProjects** [comment]
I tried publishing my page and I got the error 404 message and im trying to publish it from using the default main branch I have and it is still not showing me the page that I want to see.... do I need a root or doc folder?????

and while using GitHub desktop I dont see the page where I can view my codes/repositories in VS code either...
